src/ir_Sharp.cpp

```diff
@@ -16,6 +16,7 @@
 
 // Equipment it seems compatible with:
 //  * Sharp LC-52D62U
+//  * Sharp AC AH-A5SAY - juliussin
```

Comment:
There is a section in `ir_Sharp.h` for supported device/models etc. Can you please update that appropriately, as that gets automatically parsed into the `SupportedDevices.md` doc/page.

Thanks in advance.